ui/component/livestreamLayout/view.jsx

```diff
@@ -164,52 +165,33 @@ export default function LivestreamLayout(props: Props) {
 }
 
 const ChatModeSelector = (chatSelectorProps: any) => {
-  const { superChats, chatViewMode, setChatViewMode, activeViewers } = chatSelectorProps;
+  const { superChats, chatViewMode, setChatViewMode } = chatSelectorProps;
   const { superChatsFiatAmount, superChatsLBCAmount } = getTipValues(superChats);
 
-  const titleProps = { chatViewMode, activeViewers };
-
-  if (!superChats) {
-    return <ChatDrawerTitle {...titleProps} />;
-  }
-
-  return (
-    <Menu>
-      <MenuButton>
-        <div className="swipeable-drawer__menu">
-          <ChatDrawerTitle {...titleProps} />
-
-          <Icon icon={ICONS.DOWN} />
-        </div>
-      </MenuButton>
-
-      <MenuList className="menu__list--header">
-        <MenuItem className="menu__link" onSelect={() => setChatViewMode(VIEW_MODES.CHAT)}>
-          {__('Live Chat')}
-        </MenuItem>
-
-        <MenuItem className="menu__link" onSelect={() => setChatViewMode(VIEW_MODES.SUPERCHAT)}>
-          <div className="recommended-content__toggles">
-            <CreditAmount amount={superChatsLBCAmount || 0} size={8} /> /
-            <CreditAmount amount={superChatsFiatAmount || 0} size={8} isFiat /> {__('Tipped')}
-          </div>
-        </MenuItem>
-      </MenuList>
-    </Menu>
-  );
-};
-
-const ChatDrawerTitle = (titleProps: any) => {
-  const { chatViewMode, activeViewers } = titleProps;
-
   return (
-    <div className="swipeable-drawer__title-menu">
-      <span className="swipeable-drawer__title">
-        {chatViewMode === VIEW_MODES.CHAT ? __('Live Chat') : __('HyperChats')}
-      </span>
-
-      {activeViewers && (
-        <span className="swipeable-drawer__subtitle">{__('%view_count% viewers', { view_count: activeViewers })}</span>
+    <div className="chat-card--drawer-header-mode">
+      <Button
+        title={__('Live Chat')}
+        label={__('Live Chat')}
+        className={classnames(`button-toggle`, {
+          'button-toggle--active': chatViewMode === VIEW_MODES.CHAT && superChats,
+        })}
+        iconSize={18}
+        onClick={() => setChatViewMode(VIEW_MODES.CHAT)}
+      />
+      {superChats && (
+        <Button
+          title={__('Hyperchats')}
```

Comment:
Minor, and I think the tooltip isn't visible on mobile, 
but can this be changed to `HyperChats`?  It will then be consistent with the rest of the GUI, and stops the string generator from introducing a new one.